packages/webpack/test/aa.spec.js

```diff
@@ -0,0 +1,81 @@
+const test = require('ava')
+const { generateIdentifierLookup } = require('../src/buildtime/aa')
+
+test('aa - generateIdentifierLookup', (t) => {
+  const lookupTools = generateIdentifierLookup({
+    paths: [
+      '/home/user/app/app.js',
+      '/home/user/app/node_modules/leftpad/index.js',
+      '/home/user/app/node_modules/fast-json-patch/index.mjs',
+      '/home/user/app/node_modules/@ethereumjs/util/dist/index.js',
+      '/home/user/app/node_modules/@ethereumjs/util/node_modules/ethereum-cryptography/dist/index.js',
+      '/home/user/app/node_modules/fast-json-patch/module/duplex.mjs',
```

Comment:
can any of these be eliminated from the fixture without a drop in coverage?

Reply:
this fixture needs a redo, it is not testing some of the behaviors I wish it did.